src/middlewared/middlewared/plugins/auth.py

```diff
@@ -210,17 +215,7 @@ def __init__(self, *args, **kwargs):
         super(AuthService, self).__init__(*args, **kwargs)
         self.session_manager.middleware = self.middleware
 
-    @filterable(roles=['AUTH_SESSIONS_READ'])
-    @filterable_returns(Dict(
-        'session',
-        Str('id'),
-        Bool('current'),
-        Bool('internal'),
-        Str('origin'),
-        Str('credentials'),
-        Dict('credentials_data', additional_attrs=True),
-        Datetime('created_at'),
-    ))
+    @filterable_api_method(item=AuthSessionEntry, private=False, roles=['AUTH_SESSIONS_READ'])
```

Comment:
Isn’t  `private=False` a default?

Reply:
Actually, I got that backwards. `private=True` is default for `filterable_api_method`.

Reply:
That discrepancy with `@api_method` might cause a lot of confusion...

Reply:
That discrepancy with `@api_method` might cause a lot of confusion...

Reply:
OK. Will change.